lighthouse-core/report/html/renderer/details-renderer.js

```diff
@@ -130,7 +130,7 @@ class DetailsRenderer {
   }
 
   /**
-   * @param {LH.Audit.Details.LinkValue} details
+   * @param {Omit<LH.Audit.Details.LinkValue, "type">} details
```

Comment:
This only works if we're sure about the type of `LH.Audit.Details.LinkValue` not changing going forward, but if we're sure of it, might as well make it explicit so any changes to the code will have to deal with it :)

```suggestion
   * @param {{text: string, url: string}} details
```